uni-stark/src/symbolic_builder.rs

```diff
@@ -70,6 +71,7 @@ impl<F: Field> SymbolicAirBuilder<F> {
             .collect();
         Self {
             main: RowMajorMatrix::new(values, width),
+            public_values: vec![F::zero(); num_public_values],
```

Comment:
It isn't necessarily needed right now, but let's at least add a TODO to replace the zeros with symbolic expressions representing the `i`th public input

Reply:
I think I can add the SymbolicExpression entry for PublicValue(usize), but I'm unsure how to determine it's degree_multiple. Would it be just 0?

Reply:
Added a TODO. Btw, if we were to add the SymbolicExpression::PublicValue now, what would its degree_multiple be? I think it's 0, but I very unsure.

Reply:
Yeah I think 0 is right, from the perspective of the constraint system they're effectively constants